src/modules/rbd/rbd-list/rbd-list.js

```diff
@@ -82,6 +92,48 @@
         function createRbd() {
             $state.go("create-rbd");
         }
+
+        function onOpenRbdResizeModal(rbdObject) {
+            vm.resizeRBDstep = 1;
+            vm.resizeRBDtaskSubmitted = false;
+            var size, sizeAndUnit = [], clusterObj;
+            vm.resizeRbd = rbdObject;
+            if(rbdObject.size) {
+                size = $filter('bytes')(rbdObject.size);
```

Comment:
@kamleshverma1 Please use consistent quotes across the views. Please replace ' with " wherever applicable.